sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/ContiditionalsShouldStartOnNewLine.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        void Foo()
+        {
+            if (true)
+            {
+            } if (true)
+//            ^^ Noncompliant {{Move this 'if' to a new line or add the missing 'else'.}}
+//          ^ Secondary@-1
+            {
+            }
+
+
+            if (true)
+            {
+                // ...
+            }
+            else if (true)
+            {
+                //...
+            }
+
+            if (true)
+            {
+                // ...
+            }
+
+            if (true)
+            {
+                //...
+            }
+
+
```

Comment:
Add one more case like this:
```
if (true)
{
}
if (true)
{
}
```

Just to be sure we don't report when there is no new line between the 2 ifs.